docker-entrypoint.sh

```diff
@@ -139,17 +139,17 @@ if [ -n "$isLikelyRedmine" ]; then
 	# install additional gems for Gemfile.local and plugins
 	bundle check || bundle install
 
-	if [ ! -s config/secrets.yml ]; then
-		file_env 'REDMINE_SECRET_KEY_BASE'
-		if [ -n "$REDMINE_SECRET_KEY_BASE" ]; then
-			cat > 'config/secrets.yml' <<-YML
-				$RAILS_ENV:
-				  secret_key_base: "$REDMINE_SECRET_KEY_BASE"
-			YML
-		elif [ ! -f config/initializers/secret_token.rb ]; then
-			rake generate_secret_token
-		fi
+	file_env 'REDMINE_SECRET_KEY_BASE'
+	# just use the rails variable rather than trying to put it into a yml file
+	# https://github.com/rails/rails/blob/6-1-stable/railties/lib/rails/application.rb#L438
+	# https://github.com/rails/rails/blob/1aa9987169213ce5ce43c20b2643bc64c235e792/railties/lib/rails/application.rb#L484 (rails 7.1-stable)
+	if [ -n "${SECRET_KEY_BASE}" ] && [ -n "${REDMINE_SECRET_KEY_BASE}" ]; then
+		echo >&2
+		echo >&2 'warning: both SECRET_KEY_BASE and REDMINE_SECRET_KEY_BASE{_FILE} set, only SECRET_KEY_BASE will apply'
+		echo >&2
 	fi
+	: "${SECRET_KEY_BASE:=$REDMINE_SECRET_KEY_BASE}"
+
```

Comment:
Concretely, adding this to fix the 6.0 test failure (and maybe also a warning that they should save their `config/initializers/secret_token.rb` file?):
```suggestion
	if [ -n "$SECRET_KEY_BASE" ] && [ ! -f config/initializers/secret_token.rb ]; then
		rake generate_secret_token
	fi

```

Reply:
You mean `-z` there, right? (not `-n`)

Reply:
Also, do we need to explicitly `unset SECRET_KEY_BASE` if it's empty, or does it get ignored properly?  (We need to `export SECRET_KEY_BASE` otherwise as well, right?)

```bash
if [ -n "$SECRET_KEY_BASE" ]; then
	export SECRET_KEY_BASE
else
	unset SECRET_KEY_BASE
	if [ ! -f config/initializers/secret_token.rb ]; then
		# TODO warning?
		rake generate_secret_token
	fi
fi
```

Reply:
Added so 6.0 is working now.